ios/Sources/AutoLayoutView.swift

```diff
@@ -215,4 +228,44 @@ import UIKit
             return (cellFrame.minY >= boundsStart || cellFrame.maxY >= boundsStart) && (cellFrame.minY <= boundsEnd || cellFrame.maxY <= boundsEnd)
         }
     }
+
+    /// Fixes footer position along with rest of the items
+    private func fixFooter() {
+        guard !disableAutoLayout, let parentScrollView = getScrollView() else {
+            return
+        }
+
+        let isAutoLayoutEndVisible = horizontal ? frame.maxX <= parentScrollView.frame.width : frame.maxY <= parentScrollView.frame.height
+        guard isAutoLayoutEndVisible, let footer = footer() else {
+            return
+        }
+
+        let diff = footerDiff()
+        guard diff != 0 else { return }
+
+        if horizontal {
+            footer.frame.origin.x += diff
+            frame.size.width += diff
+            superview?.frame.size.width += diff
+        } else {
+            footer.frame.origin.y += diff
+            frame.size.height += diff
+            superview?.frame.size.height += diff
+        }
+    }
+
+    private func footerDiff() -> CGFloat {
+        if subviews.count == 0 {
```

Comment:
```suggestion
        if subviews.isEmpty {
```

`isEmpty` is generally much faster than `count` and more readable 🙂